Resources/Prototypes/holidays.yml

```diff
@@ -0,0 +1,369 @@
+- type: holiday
+  id: NewYear
+  name: New Year
+  beginDay: 30
+  beginMonth: December
+  endDay: 2
+  endMonth: January
+
+- type: holiday
+  id: MisterLizard
+  name: Mister Lizard's Birthday
+  beginDay: 15
+  beginMonth: January
+
+- type: holiday
+  id: GroundhogDay
+  name: Groundhog Day
+  beginDay: 2
+  beginMonth: February
+
+- type: holiday
+  id: ValentinesDay
+  name: Valentine's Day
+  beginDay: 13
+  endDay: 15
+  beginMonth: February
+
+- type: holiday
+  id: Birthday13
+  name: Birthday of Space Station 13
+  beginDay: 16
+  beginMonth: February
+
+- type: holiday
+  id: RandomKindness
+  name: Random Acts of Kindness Day
+  beginDay: 17
+  beginMonth: February
+
+- type: holiday
+  id: LeapDay
+  name: Leap Day
+  beginDay: 29
+  beginMonth: February
+
+- type: holiday
+  id: PiDay
+  name: Pi Day
+  beginDay: 14
+  beginMonth: March
+
+- type: holiday
+  id: StPatricksDay
+  name: St. Patrick's Day
+  beginDay: 17
+  beginMonth: March
+
+- type: holiday
+  id: Easter
+  name: Easter
+  shouldCelebrate:
+    !type:Computus { }
+
+- type: holiday
+  id: AprilFoolDay
+  name: April Fools Day
+  beginDay: 1
+  beginMonth: April
+
+- type: holiday
+  id: AutismAwarenessDay
+  name: Autism Awareness Day
+  beginDay: 2
+  beginMonth: April
+
+- type: holiday
+  id: CosmonauticsDay
+  name: Cosmonautics Day
+  beginDay: 12
+  beginMonth: April
+
+- type: holiday
+  id: SalyutAnniversary
+  name: Salyut 1's Launch Anniversary
+  beginDay: 19
+  beginMonth: April
+
+- type: holiday
+  id: FourTwenty
+  name: Four-Twenty
+  beginDay: 20
+  beginMonth: April
+
+- type: holiday
+  id: TeaDay
+  name: National Tea Day
+  beginDay: 21
+  beginMonth: April
+
+- type: holiday
+  id: EarthDay
+  name: Earth Day
+  beginDay: 22
+  beginMonth: April
+
+- type: holiday
+  id: AnzacDay
+  name: Anzac Day
+  beginDay: 25
+  beginMonth: April
+
+- type: holiday
+  id: LaborDay
+  name: Labor Day
+  beginDay: 1
+  beginMonth: May
+
+- type: holiday
+  id: FirefighterDay
+  name: Firefighter's Day
+  beginDay: 4
+  beginMonth: May
+
+- type: holiday
+  id: MothersDay
+  name: Mother's Day
+  beginMonth: May
+  shouldCelebrate:
+    !type:WeekdayInMonth
+    weekday: Sunday
+    occurrence: 2
+
+- type: holiday
+  id: OwlAndPussycatDay
+  name: Owl and Pussycat Day
+  beginDay: 12
+  beginMonth: May
+
+- type: holiday
+  id: InternationalPicnicDay
+  name: International Picnic Day
+  beginDay: 18
+  beginMonth: June
+
+- type: holiday
+  id: FathersDay
+  name: Father's Day
+  beginMonth: August
+  shouldCelebrate:
+    !type:WeekdayInMonth
+    weekday: Sunday
+    occurrence: 3
+
+- type: holiday
+  id: SummerSolstice
+  name: Summer Solstice
+  beginDay: 21
+  beginMonth: June
+
+- type: holiday
+  id: StonewallRiotsAnniversary
+  name: Stonewall Riots Anniversary
```

Comment:
Can we replace this with "Spirit Day"

https://www.glaad.org/spiritday

Celebrated it at my school in US when I was a kid, it's on October 17th